### PR TITLE
update links

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
update dead external links 

## Testing done
run locally and verify links:
- http://localhost:3001/education/gi-bill/transfer/
  - text: Learn how to transfer education benefits through milConnect
  - link: https://milconnect.dmdc.osd.mil/milconnect/public/faq/Education_Benefits-How_to_Transfer_Benefits
- http://localhost:3001/employment/job-seekers/family-members/
  - text: My Access Center
  - link: https://myaccess.dmdc.osd.mil/identitymanagement/authenticate.do?execution=e1s1
  - text: Joint Knowledge Online
  - link: http://jko.jten.mil/
- http://localhost:3001/life-insurance/options-and-eligibility/fsgli/
  - text: Go to milConnect (on page twice)
  - link: https://myaccess.dmdc.osd.mil/identitymanagement/authenticate.do?execution=e4s1
- http://localhost:3001/life-insurance/options-and-eligibility/sgli/
  - text: Go to milConnect
  - link: https://myaccess.dmdc.osd.mil/identitymanagement/authenticate.do?execution=e4s1
- http://localhost:3001/employment/job-seekers/interest-profiler/
  - text: job search
  - link: https://www.careeronestop.org/JobSearch/job-search.aspx
- http://localhost:3001/employment/job-seekers/less-than-one-year/
  - text: Joint Knowledge Online
  - link: http://jko.jten.mil/
- http://localhost:3001/employment/employers/support-veteran-employees/
  - text: tax credits
  - link: https://www.doleta.gov/business/incentives/opptax/
- http://localhost:3001/employment/job-seekers/federal-employment/
  - text: employment centers
  - link: https://www.careeronestop.org/LocalHelp/local-help.aspx

## Acceptance criteria
- [x] links rendered and work

## Definition of done
- [ ] ~Events are logged appropriately~
- [ ] ~Documentation has been updated, if applicable~
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: content/pages/employment/job-seekers/family-members.md
Comment (erikphansen) [inline content/pages/employment/job-seekers/family-members.md]: No httpS?

File: content/pages/employment/job-seekers/less-than-one-year.md
Comment (erikphansen) [inline content/pages/employment/job-seekers/less-than-one-year.md]: Okay, probably no httpS for this domain

**Review by erikphansen — APPROVED** [review]
How did you find these new URLs?

File: content/pages/employment/job-seekers/less-than-one-year.md
Comment (rianfowler) [reply content/pages/employment/job-seekers/less-than-one-year.md]: http://jko.jten.mil/videos/JKO_FOGO_Video_032015_final.mp4

**Comment by rianfowler** [comment]
@erikphansen I used [Xenu](http://home.snafu.de/tilman/xenulink.html) and visited on every external link that came back as a redirect. Most of these broken links didn't 404 so I had to manually verify them. 🙄

**Review by cehsu — APPROVED** [review]
LGTM, pending @DanielleSoCompany 's approval.

**Pull request merged**
